assets/aml-benchmark/components/batch-output-formatter/spec.yaml

```diff
@@ -13,13 +13,10 @@ inputs:
     type: string
     optional: True
     description: The metadata key that stores ground truth `label_key` in the response. If using azureml llama model and pass `_batch_request_metadata`, then this one can left empty or using `request_metadata`.
-  label_key:
-    type: string
-    optional: False
-    description: The key that stores the ground truth value.
-  data_id_key:
+  label_column_name:
     type: string
     optional: True
+    description: The key that stores the ground truth value.
```

Comment:
nit: We can update the description as well like you have for `batch-inference-preparer`- "The label column name.".

Reply:
changed